docs/index.md

```diff
@@ -1,16 +1,16 @@
-==========
-JupyterGIS
-==========
+# JupyterGIS
 
+```{eval-rst}
 .. jupyterlite::
    :new_tab: True
    :new_tab_button_text: Try it with JupyterLite!
+```
```

Comment:
````suggestion
```{jupyterlite}
:new_tab: True
:new_tab_button_text: Try it with JupyterLite!
```
````